tests/functional/idempotency/conftest.py

```diff
@@ -172,31 +161,18 @@ def expected_params_put_item(hashed_idempotency_key):
 
 
 @pytest.fixture
-def expected_params_put_item_compound_key_static_pk_value(hashed_idempotency_key, static_pk_value):
+def expected_params_put_item_compound_key_static_pk_value(
+    expected_params_put_item, hashed_idempotency_key, static_pk_value
+):
     return {
-        "ConditionExpression": (
-            "attribute_not_exists(#id) OR #expiry < :now OR "
-            "(#status = :inprogress AND attribute_exists(#in_progress_expiry) AND #in_progress_expiry < :now_in_millis)"
-        ),
-        "ExpressionAttributeNames": {
-            "#id": "id",
-            "#expiry": "expiration",
-            "#status": "status",
-            "#in_progress_expiry": "in_progress_expiration",
-        },
-        "ExpressionAttributeValues": {
-            ":now": {"N": stub.ANY},
-            ":now_in_millis": {"N": stub.ANY},
-            ":inprogress": {"S": "INPROGRESS"},
-        },
+        **expected_params_put_item,
```

Comment:
nice! I want to completely overhaul the way functional tests are written - get rid of fixtures and use builders for all of this error-prone boilerplate in stubs.

Migrating this over to the other PR.

Reply:
👍🏽